docs/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.md

```diff
@@ -6,9 +6,178 @@ jupyter:
+
+Read the insights gained from this plot [below](https://ploomber-sql.readthedocs.io/en/latest/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.html#id4)
+
+#### `Seaborn` Boxplot
+
+The dataset contains five clean fuel (electric and hybrid) US car manufacturers. To assesses the cleanest, a (grouped) boxplot is visualized of $CO_2$ emissions by vehicle type and transmission type. Similar to the `scatterplot()` explained above, the `hue` attribute is integrated with a `Dropdown` widget to toggle between vehicle type, transmission type, or no `hue`. The seaborn `boxplot()` function and widget building process is also similar to the `scatterplot()` above.
+
+Read the insights gained from this plot [below](https://ploomber-sql.readthedocs.io/en/latest/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.html#id5)
+
+## Launching the Dashboard Locally
+
+Whether your Jupyter Notebook is incomplete or complete, `Voila` will still be able to render it locally on the web! This will help you in prototyping your dashboard and fit it exactly to your needs. To launch the dashboard, run the following command in your terminal, given that you are in the same directory as the `dashboard.ipynb` file:
+
+```bash
+voila dashboard.ipynb
+```
+
+With that, you will see your Jupyter Notebook come to life as a dashboard!
+
+## Dashboard Insights
+
+The following insights were obtained from the dashboard and are also described in it:
+
+### `Seaborn` Bar Plots
+
+From the bar plot of fuel-only cars, we can see that the <b>number of unique car brand models</b> introduced to the Canadian automobile market had been increasing from the turn of the 21st century to the year 2005. This increasing trend then plateued and remained fairly constant until 2022, with 2015 experiencing the largest spike. On December 21, 2022, Steven Guilbeault, Canada's minister of environment and climate change, unveiled a regulation that would require increasing percentages of vehicle sales in Canada to be zero-emissions vehicles up to 100% by the year 2035[$^1$](https://www.canada.ca/en/environment-climate-change/news/2022/12/let-it-roll-government-of-canada-moves-to-increase-the-supply-of-electric-vehicles-for-canadians.html). These efforts seem to have had an immediate impact on the number of fuel-only cars introduced to the Canadian market, with 2023 experiencing a sharp decline and reaching 2003 levels.
```

Comment:
Summarize, reduce amount of text, embed visualization and widget

Reply:
amended in latest commit